CONTRIBUTING.md

```diff
@@ -125,11 +125,12 @@ are tested before being released. This allows us to catch UX regressions and put
 a human perspective into testing.
 
 Contributions that touch `cabal-install` are expected to include notes for the QA team.
-They are a description of an expected result upon calling `cabal-install` with certain parameters.
+They are a description of an expected result upon calling `cabal-install` with certain parameters,
+and should be written in the body of the ticket or PR under their own heading, like this:
 
 For instance:
 
-> ## QA Notes
+> \#\# QA Notes
```

Comment:
I looked at how it renders, and it needs another empty line in the quote (so, another line, containing just `>`) between the header and the text to render correctly.

Reply:
got it!